react/ContactsListModal/index.jsx

```diff
@@ -49,8 +49,14 @@ const mkFilter = filterStr => contacts => {
   })
 }
 
-const barStyle = {
-  height: 48
+const customStyle = {
+  bar: {
+    paddingLeft: '0.25rem',
```

Comment:
If 0.25rem is a new margin we can / should use a lot, it should be accessible from an utility classe.  

We should avoir inline style and avoid makeStyle() for only having to add a 0.25rem padding / margin...